bonus.mobile-app/CrmApp/src/components/ProjectCard.js

```diff
@@ -0,0 +1,111 @@
+import React, {useState} from 'react'
+import {
+  View,
+  Text,
+  StyleSheet,
+  TextInput,
+  Button,
+  TouchableOpacity
+} from 'react-native';
+import {Navigation} from 'react-native-navigation';
+
+
+function ProjectCard(props) {
+   
+/**
+ * Convert date - yyyy-mm-dd to dd/mm/yyyy
+ * @param {String} date 
+ * @returns new string date in format dd/mm/yyyy
+ */
+  const parseDate = (date) => {
+    return date.split(' ')[0].split('-').reverse().join('/');
+  };
+  
+  /**
+   * @param {string} words 
+   * @returns the same string with first capital letter of each word
+   */
+  function capitalizeTheFirstLetterOfEachWord(words) {
```

Comment:
this kind of functions (parseDate, capitalizeTheFirstLetterOfEachWord)
is classic helper material (parsingHelper or smth like that)